rdwatch/core/tasks/__init__.py

```diff
@@ -461,6 +467,14 @@ def collect_garbage_task() -> None:
         created__lte=timezone.now() - timedelta(hours=1)
     ).delete()
 
+    AnimationSiteExport.objects.filter(
+        created__lte=timezone.now() - timedelta(hours=6)
+    ).delete()
+
+    AnimationModelRunExport.objects.filter(
+        created__lte=timezone.now() - timedelta(hours=48)
+    ).delete()
+
```

Comment:
What do you think about adding some info in the UI indicating when these will "expire"?

Reply:
Yup.  Thanks I forgot to even mention that in the videos.  It could be added to the download data page (simple calculation of the created time and the now time).

Reply:
I added an expired date to the Download listing as well as a brief sentence indicating downloads will be removed (6 hours for individual sites, 48 hours for modelruns).